docs/docs/reference/metaprogramming/macros-spec.md

```diff
@@ -239,4 +239,4 @@ downright pleasant. A simple strategy first defines the underlying quoted or unq
 values using `Expr` and `Type` and then inserts quotes and splices to make the types
 line up. Phase consistency is at the same time a great guideline
 where to insert a splice or a quote and a vital sanity check that
-the result makes sense.
\ No newline at end of file
+the result makes sense.
```

Comment:
Why do the added and removed lines look indentical?

Reply:
There was an extra space at the end of the line. If you select it you can see it.

Reply:
Strange, I can't see it when selecting the line. Also, Github highlights changes in spaces by default.